src/LP/LP.jl

```diff
@@ -30,8 +30,60 @@ end
 #
 # ==============================================================================
 
+
+const MAX_LENGTH = 255
+# 16 for lp_solve 5.0: http://lpsolve.sourceforge.net/5.0/CPLEX-format.htm
+# 255 for CPLEX 12.5: https://www.ibm.com/support/knowledgecenter/SS9UKU_12.5.0/com.ibm.cplex.zos.help/FileFormats/topics/LP.html
+const START_REG = r"^([\.0-9eE])"
+const NAME_REG = r"([^a-zA-Z0-9\!\"\#\$\%\&\(\)\/\,\.\;\?\@\_\`\'\{\}\|\~])"
+
+function verifyname(name::String)
+    if length(name) > MAX_LENGTH
+        return false
+    end
+    m = match(START_REG, name)
+    if m !== nothing
+        return false
+    end
+    m = match(NAME_REG, name)
+    if m !== nothing
+        return false
+    end
+    return true
+end
+
+function correctname(name::String)
```

Comment:
This pattern
```julia
if !verifyname(name)
    name = correctname(name)
end
```
is prevalent, so `correctname` should probably be called `sanitized_name(name::String)`, which checks `verifyname` (which should be renamed to `is_valid_name`) and if true, returns the original name, otherwise, it returns the sanitized name.
Then we can just have `name = sanitized_name(name)`.